pkg/pillar/cmd/downloader/downloader.go

```diff
@@ -398,12 +398,6 @@ func handleModify(ctx *downloaderContext, key string,
 		status.ImageID, status.RefCount, config.RefCount,
 		status.Expired, status.Name)
 
-	if config.IsContainer != status.IsContainer {
-		log.Infof("handleModify: Setting IsContainer to %t for %s",
-			config.IsContainer, status.ImageID)
-		status.IsContainer = config.IsContainer
```

Comment:
Does this mean we should actually remove IsContainer from DownloaderStatus?

Reply:
Yes, we can remove it from downloader config and status.